app/experimenter/nimbus-ui/src/components/Summary/index.test.tsx

```diff
@@ -126,6 +126,7 @@ describe("Summary", () => {
         NimbusExperimentPublishStatus.REVIEW,
         {
           changelogMessage: CHANGELOG_MESSAGES.REQUESTED_REVIEW_END,
+          status: NimbusExperimentStatus.LIVE,
           statusNext: NimbusExperimentStatus.COMPLETE,
```

Comment:
FWIW we should end up using shared common props for flows when we do #5851. I didn't pull in [the mocks from PageSummary](https://github.com/mozilla/experimenter/pull/5852/files#diff-a22e33c18e87900a21ac78b38246317600edd36c1cc9885b6d322d4e1fb170f5R114) since we probably need to pull those shared props into a separate file at a higher level which would require some more refactoring and make this PR even longer, plus that's what #5851 is for.